expr/functions/linearRegression/function.go

```diff
@@ -45,9 +45,9 @@ func (f *linearRegression) Do(ctx context.Context, e parser.Expr, from, until in
 	for _, a := range arg {
 		r := a.CopyLink()
 		if len(e.Args()) > 2 {
-			r.Name = fmt.Sprintf("linearRegression(%s,'%s','%s')", a.GetName(), e.Arg(1).StringValue(), e.Arg(2).StringValue())
+			r.Name = "linearRegression(" + a.GetName() + ",'" + e.Arg(1).StringValue() + "','" + e.Arg(2).StringValue() + "')"
```

Comment:
Also move e.Arg(N).StringValue() outside a cycle/. But not blocking change, may be for simplify PR refactoing we can merge without this.